CliTool/Commands/EntityTypeCommand.cs

```diff
@@ -20,6 +22,39 @@ public EntityTypeCommand()
             };
 
             Add(nameOption);
+
+            this.SetHandler((filePathOptionValue, nameOptionValue) =>
+            {
+                InvokeCommand(filePathOptionValue, nameOptionValue);
+            },
+            Utils.FilePathOption, nameOption);
+        }
+
+        private static void InvokeCommand(string filePath, string name)
+        {
+            if (filePath == null)
+            {
+                Console.WriteLine("File path has not been set.");
+                Console.WriteLine($"Ensure you add the option --file \"/path/to/file.xml\"");
+
+                return;
+            }
+
+            if (name == null)
+            {
+                Console.WriteLine($"Ensure you add the option --name");
+
+                return;
+            }
+
+
+            CsdlXPath csdlXPath = new CsdlXPath(filePath);
+            EntityType entityType = csdlXPath.GetEntityType(name);
+
+            Console.WriteLine("***Entity Type***");
+            Console.WriteLine($"{entityType.Name}\n");
```

Comment:
Can we have the name and key in one line. Something like

`EntityType Name: Person, Key: UserName`